internal/e2e/common/issue.go

```diff
@@ -0,0 +1,104 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_common
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/cloudoperators/heureka/internal/api/graphql/graph/model"
+	util2 "github.com/cloudoperators/heureka/pkg/util"
+
+	"github.com/machinebox/graphql"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+type Issue struct {
```

Comment:
Why do we need to redefine issue here?

Reply:
This is created only to fold parameters. Using Issue from entity would be confusing because it could give to the user false impression that you can set up other members and they will be used/set correctly inside the function (for example id or list of issue variants).